lib/get-primary-value.js

```diff
@@ -7,6 +11,11 @@ module.exports = function getPrimaryValue (array) {
     return null;
   }
 
-  const primary = array.find((d) => d.primary);
```

Comment:
I try to avoid array.find as this file is used on the front end and I don't think the find method was supported on every browser